packages/office-ui-fabric-react/src/components/DetailsList/DetailsList.types.ts

```diff
@@ -245,6 +245,11 @@ export interface IDetailsListProps extends React.Props<DetailsList>, IWithViewpo
    * Whether or not the selection zone should enter modal state on touch.
    */
   enterModalSelectionOnTouch?: boolean;
+
+  /**
```

Comment:
I don't think this is necessary. The parent component can already use `componentRef` to get an `IDetailsList`, and its own `componentWillUnmount` hook to detect unmounting.